### PR TITLE
About Page added

### LINKED ISSUE #7: About Page for the website
The website should have an about page to explain what it is about.

### PR DESCRIPTION
**Pull request by prachiagrawal269**
fixed #7 
We have created the about page for the website.

@hkasera Kindly review the pull request
Attached screen shot of the same:
screenshot from 2016-03-31 19 50 35

### CODE REVIEW

**Comment by prachiagrawal269** [comment]
@hkasera Please review the PR.
Thank you

File: about.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!doctype html>
```
Comment (hkasera) [inline about.html]: Thanks for formatting the file! :+1:

File: about.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            About
+             
+        </div>
+        
+    </h1>
+    <div class="fclear"></div>
+</div>
+
+<div >
+            <h1 style= "padding: 0.8em; background-color: #E5E4E2;">About Us</h1>
```
Comment (hkasera) [inline about.html]: Why inline CSS?

File: about.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            
+ </div>
+<div >
+    <p style="font-size:20px; padding:2em;" >
```
Comment (hkasera) [inline about.html]: Inline CSS should be avoided.

File: app/routes/apiroutes.js [hunk 4]
```diff
@@ -26,5 +26,7 @@ module.exports = function(self){
     self.app.get('/get/campaign/:id/minimal' , function(req, res) {
         Campaign.getCampaignMinimalDetails(req, res);
     });
+
+
```
Comment (hkasera) [inline app/routes/apiroutes.js]: Always check for unwanted changes and remove them before submitting a PR!

**Comment by hkasera** [comment]
Please make the changes as commented :smiley:

File: about.html (hunk 2 above)
Comment (prachiagrawal269) [inline about.html]: We tried keeping the styling content into a separate CSS file and including that stylesheet into the HTML file, but that wasn't working for us. So we moved on with inline CSS.

**Comment by prachiagrawal269** [comment]
Suggested changes made.
Thanks for the suggestions.

File: about.html (hunk 2 above)
Comment (hkasera) [inline about.html]: Can you explain what was the issue? You should first try to figure out why an issue is happening rather than looking for a workaround.

**Comment by hkasera** [comment]
How did you test this code?

**Comment by prachiagrawal269** [comment]
We didn't test our code against specific test-cases as such. We simply checked for the output corresponding to the changes made (as it is front-end part). 
Are you referring to "HTML validator" or is there any specified testing framework to test our code?

**Comment by hkasera** [comment]
Testing a front end code involves testing in various browsers and in various resolutions. You should check if your UI looks the same across all latest browser versions. HTML validator is a good source to catch unintentional errors.

**Comment by hkasera** [comment]
@prachiagrawal269 Why did you reopen this? Have you made any changes in this PR?

**Comment by prachiagrawal269** [comment]
I closed it by mistake. No further changes have been made after that.

**Comment by hkasera** [comment]
Please fix the issues and let me know or else mention if you are not working on it :)

**Comment by prachiagrawal269** [comment]
@hkasera  Sorry, but we are not working on this issue any more.

**Comment by hkasera** [comment]
Sure, I will close the PR in that case :)

**Pull request closed without merging**